src/config/field-overrides/dataSet.js

```diff
@@ -1,7 +1,98 @@
+import React from 'react';
 import OrganisationUnitTreeMultiSelect from '../../forms/form-fields/orgunit-tree-multi-select';
 import DataSetElementField from './data-set/DataSetElementField.component';
 import DataInputPeriods from './data-set/DataInputPeriods.component';
 import PeriodTypeDropDown from '../../forms/form-fields/period-type-drop-down';
+import Checkbox from '../../forms/form-fields/check-box';
+import {RadioButton, RadioButtonGroup} from "material-ui/RadioButton";
+import log from "loglevel";
+
+class CustomTestComponent extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            renderAsTabs: props.value,
```

Comment:
I don't think it's necessary to add `renderAsTabs` to the state. As long as we call `props.onChange` we should be able to just use the `props.value` for this.

Reply:
good point!